### PR TITLE
Add packit.yaml file

### PR DESCRIPTION
**Pull request by girijaasoni**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: hammer_cli_foreman_leapp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-hammer_cli_foreman_leapp
+
+upstream_tag_template: "{version}"
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec -O rubygem-hammer_cli_foreman_discovery.spec"
```
Comment (ekohl) [inline .packit.yaml]: discovery -> leapp

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        additional_modules: "foreman-devel:el9"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
+    module_hotfixes: true
+
+srpm_build_deps:
+  - wget
+  - rubygems
+Footer
```
Comment (lachmanfrantisek) [inline .packit.yaml]:
```suggestion
```

Hello! Looks like the YAML parser does not like this part.

```
"  parser says\n  in "<unicode string>", line 47, column 1:\n    \n    ^\n  could not find expected \':\' while scanning a simple key"
```

You can always try to validate the config locally, using a `packit validate-config` command (https://packit.dev/docs/cli/validate-config).

If there is anything we can help, [let us know](https://packit.dev/#contact).

František from the Packit team

Comment (stejskalleos) [inline .packit.yaml]: Looks like a forgotten copy-paste from other docs ...

File: .packit.yaml (hunk 1 above)
Comment (girijaasoni) [reply .packit.yaml]: Thank you so much @ekohl , My bad, I was referencing the Discovery PR. I have corrected it and it seems to build now :)

File: .packit.yaml (hunk 2 above)
Comment (girijaasoni) [reply .packit.yaml]: Thank you! :)

File: .packit.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+files_to_sync:
+    - hammer_cli_foreman_leapp.gemspec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: hammer_cli_foreman_leapp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-hammer_cli_foreman_leapp
+
+upstream_tag_template: "{version}"
```
Comment (stejskalleos) [inline .packit.yaml]: `v{version}`

File: .packit.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: hammer_cli_foreman_leapp.gemspec
```
Comment (stejskalleos) [inline .packit.yaml]: Is this correct?

**Review by stejskalleos — CHANGES_REQUESTED** [review]

File: .packit.yaml (hunk 4 above)
Comment (stejskalleos) [reply .packit.yaml]: In https://github.com/theforeman/hammer-cli-foreman-ansible/blob/master/.packit.yaml is something different

File: .packit.yaml (hunk 3 above)
Comment (girijaasoni) [reply .packit.yaml]: it looks like "{version}" in reference with this [change](https://github.com/theforeman/hammer-cli-foreman-discovery/pull/56/commits/14f9dfe863b57bee952c80e6ee871a3bfc7262ca)

Comment (stejskalleos) [reply .packit.yaml]: That's weird, we use `vX.Y.Z` format, see the [hammer-ansible](https://github.com/theforeman/hammer-cli-foreman-ansible/blob/master/.packit.yaml)

^^ @evgeni what's the correct approach?

Comment (ekohl) [reply .packit.yaml]: Looking at https://github.com/theforeman/hammer-cli-foreman-leapp/tags this repository has prefixed tags with `v` while https://github.com/theforeman/hammer-cli-foreman-discovery/tags doesn't use that prefix.
```suggestion
upstream_tag_template: "v{version}"
```

Comment (girijaasoni) [reply .packit.yaml]: Thanks, updated :)

**Comment by ekohl** [comment]
Packit fails because https://github.com/theforeman/hammer-cli-foreman-leapp/commit/498e631046a744f0d607fb4b83b1178908d81aeb introduced new files that aren't in the `%files` section. I'd suggest to release a new version that includes it and then check back on this.

**Review by lachmanfrantisek — APPROVED** [review]
Looks fine to me. Even the SRPM build succeded so we are probably fine from the Packit's point of view.

**Review by ekohl — APPROVED** [review]
The config itself looks good.

**Comment by stejskalleos** [comment]
@ekohl packaging PR: https://github.com/theforeman/foreman-packaging/pull/10535

**Review by evgeni — APPROVED** [review]

**Pull request merged**